pkg/cmd/marshal.go

```diff
@@ -74,28 +89,44 @@ resulting data.`)
 		"right-delimiter",
 		"}}",
 		"Delimiter to use when parsing templates for substitutions")
-}
+	deprecateFlag(cmd, "as-bash-array", "use --output bash-array")
```

Comment:
Thought we made these hidden to not pollute stderr?

Reply:
they are _MarkHidden_, see the `deprecateFlag` method... it has an explanitory comment...  i kinda thought we are _deprecating_ them in the abstract sense of deprectation (as opposed to the _tied to cobra.cmd implementation_)